shaders/pbr-frag.glsl

```diff
@@ -302,6 +305,11 @@ void main()
   color = mix(color, vec3(perceptualRoughness), u_ScaleDiffBaseMR.w);
   #endif
 
+  #ifdef HAS_ALPHAMASK
+  if (baseColor.a <= u_AlphaMask) discard;
+  else baseColor.a = 1.0;
```

Comment:
Please always use braces, especially since this is reference code.

```
if (baseColor.a <= u_AlphaMask) {
    discard;
}
baseColor.a = 1.0;
```

Also I took out that `else`, I believe it's redundant with that `discard` there.